docs/useful-tips-on-carbondata.md

```diff
@@ -161,6 +162,34 @@ excessive memory usage.
 1 million.
 
 
+## Optimizing Mass Data Loading
```

Comment:
How about  using this description: Configuration for Optimizing Data Loading performance for Massive Data